wellregistry/registry/admin/monitoring_location.py

```diff
@@ -29,6 +29,19 @@ class MonitoringLocationAdminForm(ModelForm):
     """
     Registry admin form.
     """
+
+    # pylint: disable=E1101
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.user.is_superuser:
+            super().__init__(*args, **kwargs)
+        else:
+            super().__init__(*args, **kwargs)
```

Comment:
Is it necessary to call `super().__init__` two times here?

Reply:
Did not clean up after myself. Thanks for spotting this.